build.sbt

```diff
@@ -92,6 +85,9 @@ lazy val standardSettings = Seq(
 //   Wart.ExplicitImplicitTypes - see mpilquist/simulacrum#35
 //   Wart.Nothing
 //   Wart.Throw
+//   Wart.NoNeedForMonad - crashes with a match error after dependency version upgrades
+//     scala.MatchError: (_1: A, _2: T[F])(A, T[F])((a @ _), (tf @ _)) (of class scala.reflect.internal.Trees$Apply)
+//       at org.wartremover.warts.NoNeedForMonad$$anonfun$1$$anonfun$apply$1.apply(NoNeedForMonad.scala:18)
```

Comment:
Not necessarily in this PR, but we should be able to switch to `Warts.allBut` now (as we did in Quasar), which means we track the disabled ones explicitly, rather than in comments.